vcl_templates/www.vcl.erb

```diff
@@ -248,6 +248,17 @@ sub vcl_recv {
     set req.http.original-url = req.url;
   }
 
+  if (req.url.path ~ "(?i)^/register-to-vote-armed-forces") {
```

Comment:
As all paths have a common format, we can compress this `if` block down to a single clause, and use the regex captures to dictate the canonical path to set:

```
if (req.url.path ~ "(?i)^/register-to-vote(-armed-forces|-crown-servants-british-council-employees)?") {
  set req.url = std.tolower(re.group.0);
}
```

[Here's a demo](https://fiddle.fastlydemo.net/fiddle/48443bdc) using the most-excellent Fiddle tool to show it works.